UtahModules/Sources/UtahSharedContext/SourceFirestoreManager.swift

```diff
@@ -117,7 +117,6 @@ public class FirestoreManager: ObservableObject {
 //            }
 //        }
 //    }
```

Comment:
What exactly was changed here?

Reply:
Nothing, just a new line... needed to upload a change to trigger a refresh on the tests

Reply:
The tests were failing for the same reason last night (no questionnaire), but it works locally on my computer now so was hoping that if I commit this, main will work too